docs/resources/mapreduce_cluster.md

```diff
@@ -299,8 +299,11 @@ The following arguments are supported:
 * `enterprise_project_id` - (Optional, String, ForceNew) Specifies a unique ID in UUID format of enterprise project.
   Changing this will create a new MapReduce cluster resource.
 
-* `eip_id` - (Optional, String, ForceNew) Specifies the EIP ID which bound to the MapReduce cluster. Changing this will
-  create a new MapReduce cluster resource.
+* `public_ip` - (Optional, String, ForceNew) Specifies the EIP address which bound to the MapReduce cluster.
```

Comment:
why users need to set both eip id and address? you can fetch the address according the provided id yoursef